src/lib/checkVersion.ts

```diff
@@ -26,39 +29,23 @@ export class CheckVersion {
     private fs: typeof fsPromises
   ) {}
 
-  async checkVersion(
+  async checkVersion({
+    location,
+    ghToken,
+    failOnSameVersion,
+    manager,
+  }: {
     location: string,
     ghToken: string,
-    failOnSameVersion: boolean
-  ) {
-    let packageJson: {version: string} | null = null
-    let packageLockJson: any
+    failOnSameVersion: boolean,
+    manager: ManagerType,
+  }) {
+
     let newestTag: string | undefined = undefined
     let sortedTaggedVersions: Tag[] = []
 
     try {
-      //get all files in a location
-      const getFiles = new GetFiles(this.fs)
-      const filtered = await getFiles.getFiles(location)
-
-      //read and assign files
-      for (const file of filtered) {
-        const contents = await this.fs.readFile(location + file, 'utf8')
-        const jsonData = JSON.parse(contents)
-        if (file.indexOf('lock') != -1) {
-          packageJson = packageParser.parse(jsonData)
-        } else {
-          packageLockJson = packageParser.parse(jsonData)
-        }
-      }
-
-      if (packageJson === null) {
-        //change to set failed
-        return
-      }
-
-      this.compareVersions(packageJson['version'], packageLockJson['version'])
-
+      const _version: string = await this.getVersion(manager, location) 
```

Comment:
why `_`. This is a local variable so just `version`?

Reply:
it has been removed, a leftover.